code/include/playfab_c/PlayFabClientApi_c.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "httpClient/pal.h"

```

Comment:
consider breaking header into logical groups so its easier to read once its full of xml doc comments and so on

Reply:
Now broken up

---
In reply to: [591722497](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r591722497) [](ancestors = 591722497,590793685)